src/Renderer/Shader/TileFS.glsl

```diff
@@ -122,21 +122,25 @@ void main() {
                             textureIndex,
                             projWGS84 ? vUv_WGS84 : uvPM);
 
-                        if (layerColor.a > 0.0) {
+                        bool isNotTransparent = layerColor.a > 0.0;
```

Comment:
`isNotTransparent` means `isVisible` right?

Reply:
yes i changed to `isVisible`